core/modules/widgets/action-listops.js

```diff
@@ -1,3 +1,5 @@
+const { makeTiddlerIterator } = require("../wiki");
```

Comment:
Do we allow `const` now?

Reply:
Wow!! No it's much weirder than that – I didn't (intentionally) write that line of code at all,  as far as I can figure out, it was automatically inserted by VSCode and/or GitHub Copilot.

Reply:
I've pushed a fix in 681f22b66d1e37c1f4bbaf900ed41b3036e8ae81

Reply:
hihi, .. Yea I did disable Copilot again. It did add too much unwanted code and the "autocomplete feature" was constantly interrupting my thought process while programming.

Reply:
It's equally terrifying and hilarious. It does quite well for the kinds of things I might otherwise have to google, and sometimes thrills me by suggesting very TW-specific idioms, like `$tw.utils.stringifyList()`